### PR TITLE
Adding project setup

### PR DESCRIPTION
**Pull request by hpbl**
Created react project, added linter, pre-commit hooks (husky) to enforce linting, and a readme describing how to run and contribute.

solves #1

### CODE REVIEW

**Comment by hpbl** [comment]
@rrbarioni check the readme, if it makes sense, and if it works on your machine. And see if you can understand what is going on with the dependencies. If you have any questions let me know.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,71 @@
...
+npm install
+```
+
+4. Run the project:
+```sh
+npm start
+```
+
+5. If everything goes according to plan you should see the app running on http://localhost:3000
+
```
Comment (rrbarioni) [inline README.md]: Add a linting topic;
Add `language-babel`;
Change default line ending to LF at `line-ending-selector`;

File: README.md [hunk 2]
```diff
@@ -1,2 +1,71 @@
...
+
+## Release History
+
+* 0.0.1
+    * Work in progress
+
+
+## Contributors
+
```
Comment (rrbarioni) [inline README.md]: Contributor links to profiles (@hpbl and @rrbarioni) are broken

**Comment by hpbl** [comment]
@rrbarioni I've just made the requested changes, when you have the time please resume the review, and if everything is fine merge the pull request.

**Pull request merged**